site/_docs/reference.md

```diff
@@ -2780,7 +2780,7 @@ In the following:
 | m | expr1 <=> expr2                                | Whether two values are equal, treating null values as the same, and it's similar to `IS NOT DISTINCT FROM`
 | p | ACOSD(numeric)                                 | Returns the inverse cosine of *numeric* in degrees as a double. Returns NaN if *numeric* is NaN. Fails if *numeric* is less than -1.0 or greater than 1.0.
 | * | ACOSH(numeric)                                 | Returns the inverse hyperbolic cosine of *numeric*
-| o s | ADD_MONTHS(date, numMonths)                  | Returns the date that is *numMonths* after *date*
+| o s h | ADD_MONTHS(date, numMonths)                  | Returns the date that is *numMonths* after *date*
```

Comment:
This function seems not being added above, please take a look and also make sure the reference's changing consistent with what in SqlLibraryOpetators.

Reply:
Had remove this function.